tests/TextTest.php

```diff
@@ -1,12 +1,13 @@
-<?php
+<?php declare(strict_types=1);
 namespace phootwork\lang\tests;
 
 use phootwork\lang\ArrayObject;
 use phootwork\lang\tests\fixtures\Replace;
 use phootwork\lang\tests\fixtures\Search;
 use phootwork\lang\Text;
+use PHPUnit\Framework\TestCase;
 
-class TextTest extends \PHPUnit_Framework_TestCase {
+class TextTest extends TestCase {
 
 	public function testToText() {
```

Comment:
This `testToText` method has many different behavior tests. Consider trying splitting this to sliced tests.

For example, this should be sliced to `testToTextOnStartWithString`, `testToTextOnstartsWithIgnoreCase` and so on.

Reply:
Done